contact-us.php

```diff
@@ -0,0 +1,67 @@
+<?php include 'includes/header.php'; ?>
+<h1 class="bottom-spacer">Contact Us</h1>
+<h2>FAQ</h2>
+<div class="accordion-component bottom-spacer">
+    <button class="accordion">
+        <h3>My Account</h3>
+        <div class="icons">
+            <span style="font-size: $font-small; color: $secondary-color;">
```

Comment:
Why is this `font-size: $font-small; color: $secondary-color;` here? what does it do?

Reply:
styles the icon

Reply:
That code isn't actually doing anything, if you want to style these span tags you should do it in the sass file. 

Note: in order to do inline styling like you're currently doing you'd need to user raw css and not with sass variables as you've done, although inline styling is not advised all together.